### PR TITLE
Df-26: Created skill table for admin displaying all the skills and change their status

### PR DESCRIPTION
**Pull request by AyushGiri190**
Created skill table for admin to create skill and change its status.
path :
src/app/skill
src/components/skill
image
image

### CODE REVIEW

File: src/components/Domain/domainTable.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ const DomainTable: React.FC = () => {
     setLoading(true);
     setNoData(false); // Reset noData state before fetching
     try {
+      //const response = await axiosInstance.get("/domain/all/admin");
```
Comment (dehixorg) [inline src/components/Domain/domainTable.tsx:52]: please remove this comment

**Comment by AyushGiri190** [comment]
> Title says "Created skill table for admin displaying all the skills and change their status" but in file changes very minor changes are reflected. How does it work? and ticket is in backlog

Path to the files are mentioned 
The current change is just removal of a comment

**Comment by Divesh-singh** [comment]
Where are the changes to create skill table for admin? 
If these changes already merged then the tittle is very wrong Please Update title to accurately display what is being done in PR

**Comment by AyushGiri190** [comment]
> Where are the changes to create skill table for admin? If these changes already merged then the tittle is very wrong Please Update title to accurately display what is being done in PR

There is a confusion sir please correct me if I am wrong 
Yesterday the pr got reviewed and a change was to be made a comment was to be deleted. I resolved the comment.
I think I need not to create a new pr for that if my pr I not merged?

**Pull request merged**